CSS/layout.css

```diff
@@ -0,0 +1,81 @@
+/*******  CSS/LAYOUT ********/
+
+body {
+  box-sizing: border-box;
+  width: 85%;
```

Comment:
You may want to add a max-width (in px or em) in addition to width (in %) for super large screens